website/views.py

```diff
@@ -339,96 +405,115 @@ def new_question(request):
+            try:
+                sender_name = "FOSSEE Forums"
+                sender_email = "forums@fossee.in"
+                subject = "FOSSEE Forums - {0} - New Question"\
+                          .format(question.category)
+                to = (question.category.email, 'forum-notifications@fossee.in')
+                url = settings.EMAIL_URL
+
+                message = """
+                    The following new question has been posted in the FOSSEE Forum: <br>
+                    <b> Title: </b>{0}<br>
+                    <b> Category: </b>{1}<br>
+                    <b> Link: </b><a href="{3}">{3}</a><br>
+                    <b> Question : </b>{2}<br>
+                """.format(
+                    question.title,
+                    question.category,
+                    question.body,
+                    'http://forums.fossee.in/question/'+str(question.id),
+                )
+                email = EmailMultiAlternatives(
+                    subject, '',
+                    sender_email, to,
+                    headers={"Content-type": "text/html;charset=iso-8859-1"}
+                )
+                email.attach_alternative(message, "text/html")
+                email.send(fail_silently=True)
+            except:
+                pass
             return HttpResponseRedirect('/')
         else:
-             context.update(csrf(request))
-             category = request.POST.get('category', None)
-             tutorial = request.POST.get('tutorial', None)
-             context['category'] = category
-             context['tutorial'] = tutorial
-             context['form'] = form
-             context['toolbox'] = toolbox
-             return render(request, 'website/templates/new-question.html', context)
+            context.update(csrf(request))
+            category = request.POST.get('category', None)
+            tutorial = request.POST.get('tutorial', None)
+            context['category'] = category
+            context['tutorial'] = tutorial
+            context['form'] = form
+            context['toolbox'] = toolbox
+
+            return render(request, 'website/templates/new-question.html',
+                          context)
     else:
-       
+
         category = request.GET.get('category')
         if category == 12:
             toolbox = True
         tutorial = request.GET.get('tutorial')
-        form = NewQuestionForm(category=category,tutorial = tutorial)
+        form = NewQuestionForm(category=category, tutorial=tutorial)
         context['category'] = category
         context['tutorial'] = tutorial
         context['toolbox'] = toolbox
-        
+        context['FossCategory'] = FossCategory.objects.get(id=category)
+        print context['FossCategory'].default_tags.all()
     context['form'] = form   
     context.update(csrf(request))
     return render(request, 'website/templates/new-question.html', context)
 
 
 # return number of votes and initial votes
```

Comment:
This should be a docstrings. Please refer to Python docs and add this info as a docstring for this function